### DISCUSSION TITLE
Need help with running a restore

### DISCUSSION THREAD

**Discussion by kiankyars** [GAM-team/got-your-back#339] [Q&A]
I am trying to run a restore based off of an .mbox file that was obtained through google takeout; although I am having trouble with the syntax, I believe the following is the command that should be run
 "gyb --email youremail@gmail.com --action restore-mbox --local-folder "c:\my_gmail_backup""
However, it says command not found when I attempt to execute it, I assume that I am not entering the local folder part correctly as I checked to make sure I typed my email correctly. If I have my .mbox file on my mac desktop, what should I write in that field order for it to work?

Thanks!

**Comment by jorge-gbs** [comment]
You're either probably not running the gyb command from the folder where it is installed or you're referencing the Windows File path on a Mac. You need to use the Mac file path. Here are some quick instructions on how to get the mac file path: https://macpaw.com/how-to/get-file-path-mac

**Reply by kiankyars** [reply]
Thanks @jorge-gbs, I was able to find the file path, and when I enter the following: 
"gyb --email kian@gmail.com --action restore-mbox --local-folder /Users/Kian/Desktop/n.mbox"
 it still says path not found, I also tried putting quotations around the file path but to no avail. Can you please check to see if I formatted anything incorrectly, also I got this message when finishing the installation, so I believe everything is set up, but how would I know if I am not running the gyb command from the folder where it is installed?
![Screen Shot 2021-12-30 at 10 38 46 AM](https://user-images.githubusercontent.com/69437137/147775553-eb78783c-97da-4276-a85c-b6bc1fb62a77.png)
 
Thanks

**Comment by jorge-gbs** [comment]
not super familiar with Mac but in Linux, it would either run when you call "gyb <Command parameters>" or you may need to change dir to the dir where its located. Here are some instructions on how to use the terminal on Mac: https://www.macworld.com/article/221277/command-line-navigating-files-folders-mac-terminal.html

**Comment by ProgmanCsimple** [comment]
Unfortunaly, it seem you need to be a tester to use it. Google Cloud say so ("GYB n'a pas terminé la procédure de validation de Google. L'appli est en cours de test, et seuls les testeurs approuvés par le développeur y ont accès."). Otherwise, GYB is blocked  ("Accès bloqué : GYB n'a pas terminé la procédure de validation de Google").
